engine/src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -1,20 +1,14 @@
 package cromwell.server
 
-import akka.actor.{Props, ActorSystem}
+import akka.actor.ActorSystem
 import com.typesafe.config.ConfigFactory
 import cromwell.engine.backend.CromwellBackend
-import cromwell.engine.workflow.{MaterializeWorkflowDescriptorActor, WorkflowManagerActor}
 import cromwell.engine.workflow.WorkflowManagerActor
-import cromwell.instrumentation.Instrumentation._
 import collection.JavaConversions._
 
 trait WorkflowManagerSystem {
-  Monitor.start()
-
   protected def systemName = "cromwell-system"
-
   protected def newActorSystem(): ActorSystem = ActorSystem(systemName)
-
```

Comment:
Remove these lines before/after the `def`s all you want. IntelliJ ⌘⌥O is going to put them right back.